pkg/sqlite/load.go

```diff
@@ -7,15 +7,18 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/blang/semver"
 	_ "github.com/mattn/go-sqlite3"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 
+	libsemver "github.com/operator-framework/operator-registry/pkg/lib/semver"
 	"github.com/operator-framework/operator-registry/pkg/registry"
 )
 
 type sqlLoader struct {
-	db       *sql.DB
-	migrator Migrator
+	db          *sql.DB
+	migrator    Migrator
+	enableAlpha *bool
```

Comment:
by the time we get here I don't think we need a tri-state. let's just make it a `bool`?